amundsen_application/api/metadata/v0.py

```diff
@@ -494,3 +472,68 @@ def _log_get_user(*, user_id: str) -> None:
         logging.exception(message)
         payload = jsonify({'msg': message})
         return make_response(payload, HTTPStatus.INTERNAL_SERVER_ERROR)
+
+
+@metadata_blueprint.route('/user/bookmark', methods=['GET'])
+def get_bookmark() -> Response:
+    try:
+        user_id = request.args.get('user_id')
+        if user_id is None:
+            if app.config['AUTH_USER_METHOD']:
+                user_id = app.config['AUTH_USER_METHOD'](app).user_id
+            else:
+                raise Exception('AUTH_USER_METHOD is not configured')
+
+        url = '{0}{1}/{2}/follow/'.format(app.config['METADATASERVICE_BASE'], USER_ENDPOINT, user_id)
```

Comment:
we only use bookmark endpoint . But we could put a todo/note on putting the share endpoint into a constant once we introduce own/read endpoint in FE.

Reply:
Since the URL pattern requires the `user_id` to go in the middle of the string `[metadataservice]/user/[user_id]/follow`, it makes it difficult use a single constant. 

We could do something like
```
'{0}{1}/{2}/{3}'.format(app.config['METADATASERVICE_BASE'], USER_ENDPOINT, user_id, FOLLOW_ENDPOINT)
```
which seems somewhat complicated. Maybe a utility function could generate this in a better way?

This might be breaking the restful pattern but something like the following might be easier.
```
user/follow/[user_id]
user/own/[user_id]
user/read/[user_id]
```

Reply:
not sure why the following is complicated
```
'{0}{1}/{2}/{3}'.format(app.config['METADATASERVICE_BASE'], USER_ENDPOINT, user_id, FOLLOW_ENDPOINT)
```
but I agree with @danwom we should NOT go against the restful pattern.